src/models/Erc721TransactionModel.ts

```diff
@@ -0,0 +1,92 @@
+const mongoose = require("mongoose");
+const mongoosePaginate = require("mongoose-paginate");
+const Schema = mongoose.Schema;
+
+
+/**
+ * Model for a single transaction.
+ *
+ * @type {"mongoose".Schema}
+ */
+const transactionSchema = new Schema({
```

Comment:
Is there a way not to duplicate this model? Maybe add `collectibleOperations` instead?

Reply:
The reason I separated everything and even tolerated some duplicate here is that I tried not to change the existing things in order not to break them. I don't feel confident to change them and still make sure they work, given the current test coverage.

For now, keeping the model separate might help with testing ERC721 parsing in warranty period.

Reply:
It will be doing lots of extra requests, I think we need to store events for each transactions and then read them

Reply:
Yeah, the only difference is that for ERC20, we are only interested in 'Transfer' operations, and for ERC721, I added 'Approval' and 'approve'.

Did you mean that we should store all operations regardless of their types, then we can read them later when necessary?